core/src/mindustry/mod/Mods.java

```diff
@@ -787,7 +799,7 @@ public Seq<Fi> resolveDependencies(Seq<Fi> files){
 
             try{
                 Fi zip = file.isDirectory() ? file : new ZipFi(file);
-
+                if(OS.isMac) zip.child(".DS_Store").delete(); //macOS loves adding garbage files that break everything
```

Comment:
This will lead to very strange results if a zip file has a DS_Store entry in it... you can't delete those, it just closes the zip.

Reply:
I mean... I've been doing it for months with no issues.

Reply:
was a trivial fix either way